Dockerfile

```diff
@@ -21,8 +21,6 @@ RUN    apt-get update           \
             pkg-config          \
             python3             \
             python-pygments     \
-            python-recommonmark \
-            python-sphinx       \
             rapidjson-dev       \
             zlib1g-dev
 
```

Comment:
Should  we update the z3 version on the line below? It's `4.8.7` and the latest one is `4.8.10` https://github.com/Z3Prover/z3/releases/.

Reply:
I ended up just removing the manual installations of Z3, because 4.8.7 is default in Ubuntu Focal. 4.8.7 is actually best Z3 for us, later versions have knows bugs.